vendor.conf

```diff
@@ -107,12 +107,12 @@ google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 github.com/containerd/containerd 3fa104f843ec92328912e042b767d26825f202aa
```

Comment:
just wondering, why we don't keep this the same as the sha in binaires-commits?

Reply:
We want to use the official tagged releases for the binary, but the latest code for the client library, so they will be different SHAs.